README.md

```diff
@@ -54,6 +50,10 @@ turn it back off when you're done running reports, use the `coverageOff` command
 
 Sample project with scoverage in both sbt and maven - [the scoverage samples project](https://github.com/scoverage/sbt-scoverage-samples).
 
+## Notes on upgrading to version 1.6.x
+* ´coverageCleanSubprojectFiles´ key has been removed
+* scala version 2.12+ is required as a minimum
```

Comment:
If possible put a small note that this isn't an sbt-scoverage thing, but rather the scoverage support itself. Something like:
```suggestion
* scala version 2.12+ is required as a minimum (due to scoverage suppport)
```

Reply:
👍